core/mutator/entry/mutation_test.go

```diff
@@ -42,3 +46,65 @@ func TestReplaceAuthorizedKeys(t *testing.T) {
 		}
 	}
 }
+
+func TestCreateAndVerify(t *testing.T) {
+	for _, tc := range []struct {
+		old     []byte
+		pubKeys []*keyspb.PublicKey
+		signers []signatures.Signer
+		data    []byte
+		wantErr bool
+	}{
+		{
+			old:     nil,
+			pubKeys: []*keyspb.PublicKey{mustPublicKey(testPubKey1)},
```

Comment:
maybe this could be:
`pubKeys: mustPublicKeys([]string{testPubKey1}),`

feels marginally shorter?

Reply:
Yeah, thanks!